Makefile

```diff
@@ -23,74 +23,58 @@ stack-down: ## stop dsp-stack and remove the cloned dsp-api repository
 
 .PHONY: dist
 dist: ## generate distribution package
-	python3 setup.py sdist bdist_wheel
+	@rm -rf dist/ build/
+	poetry build
 
 .PHONY: upload
 upload: ## upload distribution package to PyPI
-	python3 -m twine upload dist/*
+	poetry publish
 
 .PHONY: docs-build
 docs-build: ## build docs into the local 'site' folder
-	mkdocs build
+	poetry run mkdocs build
 
 .PHONY: docs-serve
 docs-serve: ## serve docs for local viewing
 	mkdocs serve --dev-addr=0.0.0.0:7979
 
-.PHONY: install-requirements
```

Comment:
This command was only necessary for CI, but now it's simpler: `make install` does everything. And if a developer wants to work with the code, the README instructs him that he must execute `poetry install`.